cmd/terramate/cli/run.go

```diff
@@ -285,7 +311,13 @@ func (c *cli) RunAll(runStacks []ExecContext, isSuccessCode func(exitCode int) b
 					logger.Error().Err(err).Msg("failed to execute")
 				}
 
-				c.cloudSyncAfter(runContext, result.cmd.ProcessState.ExitCode(), err)
+				res := RunResult{
+					ExitCode:   result.cmd.ProcessState.ExitCode(),
+					StartedAt:  &startTime,
+					FinishedAt: result.finishedAt,
+				}
+
```

Comment:
What about logging the duration of the run here? This information can be useful for platform engineers.

```
logger.Debug().
    Int("exit_code", res.ExitCode).
    Time("started_at", res.StartedAt).
    Time("finished_at", res.StartedAt).
    TimeDiff("duration", res.FinishedAt, res.StartedAt).
    Msg("command execution finished")
```